Qubit Server/dev/src/org/labrad/qubits/QubitContext.java

```diff
@@ -560,7 +560,9 @@ public void sram_dual_block_delay(@Accepts("v[ns]") double delay) {
 				+ "microwave quadratures.  The length of the data should "
 				+ "match the length of the current SRAM block.  "
 				+ "An optional boolean specifies whether the data "
-				+ "should be deconvolved (default: true).")
+				+ "should be deconvolved (default: true)."
+                + "If deconvolve=false, the data can be specified as DAC-ready"
+                + "I and Q integers.")
```

Comment:
indentation is screwy here. I think the file as it stands uses tabs, and I'm guessing the new lines use spaces. Let's convert this file to use spaces everywhere.